Source/Tools/Flax.Build/Projects/VisualStudio/CSSDKProjectGenerator.cs

```diff
@@ -106,14 +111,14 @@ public override void GenerateProject(Project project, string solutionPath)
             csProjectFileContent.AppendLine("    <LangVersion>11.0</LangVersion>");
             csProjectFileContent.AppendLine("    <FileAlignment>512</FileAlignment>");
 
-            // Needed for Hostfxr
-            csProjectFileContent.AppendLine("    <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>");
-            csProjectFileContent.AppendLine("    <EnableDynamicLoading>true</EnableDynamicLoading>");
```

Comment:
Why was it removed?

Reply:
These values are not used by IDE, I think these were leftovers from the early prototypes of the .NET 7 support.